### PR TITLE
fix(tests): Update loadtests to cope with sign-in confirmation

### LINKED ISSUE #1828: repair ./tests/load loadtest
This was working until recently, but now I get unexplained `ClientError: Unverified account` running `make test` or `make build`. It happens sometimes but not always. This may be a problem with auth-server, but I haven't figured it out yet what's wrong there.

However, I was running with an old (0.0.9) version of FxAPy. So I updated to current 0.3.0, but now was getting `AttributeError: 'module' object has no attribute 'epoll'`, which is probably some `urllib3 v. gevent` monkey patch confusion, as noted in https://github.com/kennethreitz/requests/issues/3831.

I worked around it by hacking the virtualenv ./bin/loads-runner, to add
```
from gevent import monkey
monkey.patch_all()
```
as the first thing run there. And that let me get back to having the `ClientError: Unverified account` with 0.3.0 PyFxA. (Obviously, not a real fix).

Anyways, it would be good to get this all running properly (in PyFxA and fxa-auth-server) to be able to run the load test, to compare non-docker and docker runs in stage.

### PR DESCRIPTION
**Pull request by rfk**
This fixes #1828 by using the restmail utilities in `PyFxA` to perform session confirmation if required.  It means we hit restmail on occasion in the middle of the loadtest, but since the bulk of requests are not login related I don't think it'll make a huge difference overall.  I also took the opportunity to make the password-reset test actual codes as well, for good measure.

With this patch I can consistently get `make bench` to pass cleanly against latest stage environment.

@jrgm r?

### CODE REVIEW

File: test/load/loadtests.py [hunk 1]
```diff
@@ -268,3 +309,15 @@ def test_password_change_flow(self):
     def test_support_doc_flow(self):
         base_url = self.server_url[:-3]
         self.session.get(base_url + "/.well-known/browserid")
+
+    def _get_code_from_email(self, emailAcct, header):
+        # Due to possible race conditions, and previous test runs
+        # leaving junk in the restmail account, we look for the last
+        # matching email rather than the first.  There's still a small
+        # chance that we pull out the wrong code here.
+        for _ in xrange(10):
```
Comment (jrgm) [inline test/load/loadtests.py]: Why `_ in xrange(10)`?

Comment (rfk) [inline test/load/loadtests.py]: it's just a retry loop to wait for the email to arrive; similar logic to [1] except we want to most recent email, and we want to eventually fail out and I didn't want to sleep.

[1] https://github.com/mozilla/PyFxA/blob/master/fxa/tests/utils.py#L97

Comment (jrgm) [inline test/load/loadtests.py]: Yeah, it tries to refetch the mailbox if it couldn't find a header. I guess the magic number made me go huh?

Comment (rfk) [inline test/load/loadtests.py]: fair, I'll give it a name like `num_retries` or something to make this clearer

**Pull request merged**